src/qtism/runtime/rendering/markup/AbstractMarkupRenderingEngine.php

```diff
@@ -638,6 +643,10 @@ protected function processNode($base = ''): void
             $this->includeChoiceComponent($component, $rendering);
         }
 
+        if ($this->mustTwigFeedbackComponent($component) === true) {
+            $this->twigFeedbackComponent($component, $rendering);
```

Comment:
I propose to clear up the naming a bit. If I understand it correctly:

Does this mean if twig feedback component must be used? I propose to rename to:
if ($this->mustUseTwigFeedbackComponent($component) === true) { ... }

and set twig feedback component:
```$this->setTwigFeedbackComponent($component, $rendering);```